tests/e2e/config/bootstrap.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { setDefaultOptions } from 'expect-puppeteer';
+
+/**
+ * WordPress dependencies
+ */
+import {
+  enablePageDialogAccept,
+  setBrowserViewport,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Environment variables
+ */
+const { PUPPETEER_TIMEOUT, EXPECT_PUPPETEER_TIMEOUT } = process.env;
+
+/**
+ * Set of console logging types observed to protect against unexpected yet
+ * handled (i.e. not catastrophic) errors or warnings. Each key corresponds
+ * to the Puppeteer ConsoleMessage type, its value the corresponding function
+ * on the console global object.
+ *
+ * @type {Object<string,string>}
+ */
+const OBSERVED_CONSOLE_MESSAGE_TYPES = {
+  warning: 'warn',
+  error: 'error',
+};
+
+/**
+ * Array of page event tuples of [ eventName, handler ].
+ *
+ * @type {Array}
+ */
+const pageEvents = [];
+
+// The Jest timeout is increased because these tests are a bit slow
+jest.setTimeout(PUPPETEER_TIMEOUT || 100000);
```

Comment:
Should we still keep the increased timeout?

Reply:
Will test. Perhaps unneeded now.

Reply:
Looks like it's indeed needed because the default timeout in Jest (test runner itself) is not suitable for a complex thing like browser testing.

Reply:
Alright, thanks for checking.